Makefile

```diff
@@ -79,7 +79,7 @@ validate-shell:
 
 .PHONY: shellspec
 shellspec:
-	docker build --platform $(shell uname -m) -t shellspec-docker - < ./spec/shellspec.Dockerfile
+	docker build --platform linux/amd64 -t shellspec-docker - < ./spec/shellspec.Dockerfile
```

Comment:
do we need the --platform flag ?

Reply:
yeah I thought it was auto-detected, not sure though

Reply:
Yes just checked. We need it.

Reply:
Oh it seems we don't need it. It works in local machine without it.
Removed!